applications/matter_bridge/Kconfig

```diff
@@ -159,6 +159,10 @@ config CHIP_ENABLE_READ_CLIENT
 config NCS_SAMPLE_MATTER_PERSISTENT_STORAGE
 	default y
 
+config BRIDGE_SMART_PLUG_SUPPORT
+	bool "Allowing certain parts of the code to be compiled only when selecting a zap file from snippets"
```

Comment:
I would try to keep this description less technical and more high-level, like:

"Enables additional endpoint implementing Matter smart plug functionality"